### ISSUE TITLE
How to generate a new landmark txt file for a new photo?

### ISSUE THREAD

**Issue by be-nice-do-better** [yiranran/APDrawingGAN#24]
I put a new photo _test.png_ in folder _dataset\data\test_single_, run the command:
`python test.py --dataroot dataset/data/test_single --name formal_author --model test --dataset_mode single --norm batch --use_local --which_epoch 300 --gpu_ids -1`
got the below error:

> FileNotFoundError: [Errno 2] No such file or directory: 'dataset/landmark/ALL\\test.txt'

How to generate a new landmark txt file for this new photo?

**Comment by feitgemel** [comment]
Did you got an answer ?

**Comment by feitgemel** [comment]
Hi,
Can you elaborate what are those txt files inside the Landmark folder?
How can I generate a file for my one image ?

I took a fresh image, convert it to 512X15 , and created a mask for the head.
How to generate this landmark txt file ?

Eran

**Comment by be-nice-do-better** [comment]
No, I am using Windows OS, and one of the prerequisites is using Linux or macOS, so could be my environment issue
